packages/synapse-react-client/src/style/components/entity_finder/_details-view.scss

```diff
@@ -27,6 +26,7 @@ $row-hovered-background-color: unset;
   height: $-finder-height;
   width: 100%;
   background-color: SRC.$background-color-gray-light;
+  overflow-y: hidden;
```

Comment:
Apply `overflow-y: hidden` in all cases. This causes the overflow to be applied to the table instead of the container wrapping the table, so now the container that holds the search field does not stretch to match the width of the outer container.